packages/visitors/visitors.20180306/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "François Pottier <francois.pottier@inria.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/visitors"
+dev-repo: "https://gitlab.inria.fr/fpottier/visitors.git"
+bug-reports: "francois.pottier@inria.fr"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the ```{build}``` attribute here if you are not using ```Findlib``` ? (it avoids rebuilding the package on each upgrades/reinstalls of ocamlfind)

Reply:
OK, done. Thanks for the suggestion!